packages/components/pie-modal/test/visual/pie-modal.spec.ts

```diff
@@ -501,28 +465,28 @@ test.describe('Prop: `isFooterPinned`', () => {
 test.describe('Prop: `hasStackedActions`', () => {
     test.describe('when true', () => {
         sizes
-            .forEach((size) => {
-                test(`should display actions full width (at narrow viewports – with leading action on top) for a modal with size = ${size}`, async ({ page, mount }) => {
-                    await mount(PieModal, {
-                        props: {
-                            heading: 'This is a modal heading',
-                            hasStackedActions: true,
-                            isOpen: true,
-                            size,
-                            leadingAction: {
-                                text: 'Confirm',
-                            },
-                            supportingAction: {
-                                text: 'Cancel',
-                            },
-                        } as ModalProps,
-                    });
-
-                    const modal = page.locator(componentSelector);
-                    await expect.soft(modal).toBeVisible();
-
-                    await percySnapshot(page, `Modal - hasStackedActions = true, size = ${size}`);
-                });
-            });
+      .forEach((size) => {
+          test(`should display actions full width (at narrow viewports – with leading action on top) for a modal with size = ${size}`, async ({ page }) => {
+              const modalDefaultPage = new ModalDefaultPage(page);
+              const props: Partial<ModalProps> = {
+                  heading: 'This is a modal heading',
+                  hasStackedActions: true,
+                  isOpen: true,
+                  size,
+                  leadingAction: {
+                      text: 'Confirm',
+                  },
+                  supportingAction: {
+                      text: 'Cancel',
+                  },
+              };
+
+              await modalDefaultPage.load(props);
+
+              await expect.soft(modalDefaultPage.modalComponent.componentLocator).toBeVisible();
+
+              await percySnapshot(page, `Modal - hasStackedActions = true, size = ${size}`, percyWidths);
```

Comment:
Given the test mentions `(at narrow viewports – with leading action on top)` - I added the `percyWidths` to only test this story in mobile viewport (375px)

Reply:
Hmm I see what you mean but I think it's valuable testing the wider viewport as well just to make sure there isn't a regression and we don't show them stacked